app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt

```diff
@@ -228,6 +219,36 @@ fun FlightScreen(
                           .background(color = Color.White, shape = RoundedCornerShape(8.dp))
                           .padding(6.dp))
             }
+            var showConfirmationDialog by remember { mutableStateOf(false) }

```

Comment:
This should be moved up with the other variables

Reply:
Moved it !